src/client.rs

```diff
@@ -65,7 +68,7 @@ impl DatagoClient {
             source_type: config.source_type,
             source_config: config.source_config,
             limit: config.limit,
-            max_connections: 512,
+            max_connections: 128,
```

Comment:
more doesn't actually help long term + we often instantiate multiple clients in parallel